include/Dialects/LinalgExt/Passes.td

```diff
@@ -27,6 +27,33 @@ def LinalgExtTiling : Pass<"linalg-ext-tiling", "FuncOp"> {
   ];
 }
 
+def LinalgExtTilingToTileOp : Pass<"linalg-ext-tiling-to-tile-op", "FuncOp"> {
+  let summary = "Pass to tiling TilingInterface operations to 1 linalg_ext.tile op.";
+  let description = [{
+    Rewrite a TilingInterface operation to a linalg_ext.tile.
+
+    This transformation is meant to tile only a single dimension.
+    This is because TileOp is meant to remain a 1-D loop construct with parallel 
+    semantics. We specifically want to avoid TileOp becoming an n-D loop construct
+    in the absence of a very compelling motivation.
+    Unlike scf.for we do not need a one-shot multi-loop tiling when targeting TileOp.
+    Indeed the implicitly stacking terminator + outSlices operands bypass the 
+    traditional issues with inserting subset.insert/subset.extract op pairs.
```

Comment:
Can you specify what happens if you have multiple operations that satisfy `TilingInterface` in your program? The two interesting cases is when they are nested and when they are not nested I guess.

Reply:
The nested case is not yet a thing.

Reply:
Added some comment to explain this is experimental and not yet meant to be used with multiple ops.